Libplanet/Blockchain/BlockChain.cs

```diff
@@ -209,24 +211,29 @@ ActionEvaluator<T> actionEvaluator
             IBlockChainStates<T> blockChainStates,
             ActionEvaluator<T> actionEvaluator)
         {
+            if (store is null)
+            {
+                throw new ArgumentNullException(
+                    nameof(store), $"Given {nameof(store)} cannot be null");
```

Comment:
Would we need this message? 🤔

Reply:
Eh, I suppose `ArgumentNullException` does sufficient job in this case and we don't need to elaborate (or provide a custom message). 🙄